.github/workflows/tox-tests.yml

```diff
@@ -0,0 +1,124 @@
+name: Tox tests
+
+on: [push, pull_request]
+
+jobs:
+  py26:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Update existing dependencies
+        run: sudo apt-get update -y
+      - name: Install system dependencies
+        run: sudo apt-get install -y libkrb5-dev
+      - name: Add Python 2.6 repository
+        run: sudo add-apt-repository ppa:deadsnakes/ppa
+      - name: Install Python 2.6
+        run: sudo apt-get install -y python2.6 python2.6-dev
+      - name: Install pip
+        run: curl https://bootstrap.pypa.io/2.6/get-pip.py -o get-pip.py; sudo python2.6 get-pip.py
```

Comment:
just small question. Would not be better to use &&:
`curl https://bootstrap.pypa.io/2.6/get-pip.py -o get-pip.py && sudo python2.6 get-pip.py`

Reply:
Changed.